src/Files.App/Data/Items/DriveItem.cs

```diff
@@ -239,7 +239,7 @@ public static async Task<DriveItem> CreateFromPropertiesAsync(StorageFolder root
 				IsLocationItem = true,
 				ShowEjectDevice = item.IsRemovable,
 				ShowShellItems = true,
-				ShowFormatDrive = !(item.Type == DriveType.Network || string.Equals(root.Path, "C:\\", StringComparison.OrdinalIgnoreCase)),
+				ShowFormatDrive = !(item.Type == DriveType.Network || string.Equals(root.Path, $@"{Environment.GetEnvironmentVariable("SystemDrive")}\", StringComparison.OrdinalIgnoreCase)),
```

Comment:
It'd be better to use root.Path.StartsWith
```suggestion
				ShowFormatDrive = !(item.Type == DriveType.Network || root.Path.StartsWith($@"{Environment.GetEnvironmentVariable("SystemDrive")}", StringComparison.OrdinalIgnoreCase)),
```

Reply:
What would be the problem if I leave it?

Reply:
Having backslash at the end or not frequently becomes a problem.
Yet, you can leave as is, though.

Reply:
If there is no clear problem, I don't want to change the behavior as much as possible.

Reply:
For me it's clear since I've coped with it using https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathaddbackslasha ensuring it has backslash at the end in my C projects.
But I agree, this change is out of scope.